src/compressor/QatAccel.h

```diff
@@ -37,6 +39,14 @@ class QatAccel {
   int compress(const bufferlist &in, bufferlist &out, boost::optional<int32_t> &compressor_message);
   int decompress(const bufferlist &in, bufferlist &out, boost::optional<int32_t> compressor_message);
   int decompress(bufferlist::const_iterator &p, size_t compressed_len, bufferlist &dst, boost::optional<int32_t> compressor_message);
+
+private:
+  friend struct cached_session_t;
+  std::vector<session_ptr> sessions;
+  std::mutex mutex;
+  std::condition_variable cond;
+  bool init_called{false};
```

Comment:
nit, is `!alg.empty()` good enough for serving the same purpose?

Reply:
yes, I have replace `init_called` with `!alg_name.empty()`. Thanks